api/src/tests/index_test.rs

```diff
@@ -113,9 +115,10 @@ async fn test_cors_on_non_200_responses() {
         .header("Access-Control-Request-Headers", "Content-Type")
         .header("Access-Control-Request-Method", "GET")
         .method("GET")
-        .path("/accounts/nope/resources");
+        .path("/v1/accounts/nope/resources");
     let resp = context.reply(req).await;
     assert_eq!(resp.status(), 400);
     let cors_header = resp.headers().get("access-control-allow-origin").unwrap();
-    assert_eq!(cors_header, "*");
+    // TODO: Make an issue
```

Comment:
What's the issue? Does CORS not work?

Reply:
It works, but it's not the same as it was before. Max says it's still good though.